docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,10 @@ IBM® block storage CSI driver 1.10.0 introduces the enhancements that are detai
 
 **General availability date:** 15 July 2022
 
+## New dynamic host definition
+
+IBM® block storage CSI driver 1.10.0 enables users to not have the need to statically define dynamic objects on the storage in advance. The `HostDefiner` custom resource handles changes in the Kubernetes clusters that relate to the host definition feature and applies them to the relevant clusters.
```

Comment:
how about:
```suggestion
IBM® block storage CSI driver 1.10.0 can be deployed with a new host definer component, which dynamically defines nodes connectivity on storage for the user.
```

Reply:
and optionally - `, eliminating the need for manual static host definitions`

Reply:
updated